azurerm/resource_arm_app_service_test.go

```diff
@@ -2412,3 +2442,41 @@ resource "azurerm_app_service" "test" {
 }
 `, rInt, location, rInt, rInt, tlsVersion)
 }
+
+func testAccAzureRMAppService_corsSettings(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_app_service_plan" "test" {
+  name                = "acctestASP-%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  sku {
+    tier = "Standard"
+    size = "S1"
+  }
+}
+
+resource "azurerm_app_service" "test" {
+  name                = "acctestAS-%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  app_service_plan_id = "${azurerm_app_service_plan.test.id}"
+
+  site_config {
+    cors {
+      allowed_origins = [
+        "http://www.contoso.com",
+        "www.contoso.com",
+        "contoso.com"
```

Comment:
could we also include an updated test (we can apply the config multiple times see https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_app_service_test.go#L302-L315) that flips the order of these to ensure they're still set correctly?